### PR TITLE
Inmobi - Mopub GDPR

### PR DESCRIPTION
**Pull request by kshtgarg21**

### CODE REVIEW

File: InMobi/InMobiAdapterConfiguration.m [hunk 1]
```diff
@@ -22,6 +22,8 @@ - (NSString *)adapterVersion {
 }
 
 - (NSString *)biddingToken {
+    [InMobiAdapterConfiguration updatePartnerGDPRConsent];
```
Comment (p158276) [inline InMobi/InMobiAdapterConfiguration.m:25]: We call the bidding Token everytime when ad request happen.. You don't need to update GDPR here as you already call the updatePartnerGDPRConsent before the ad request.

File: InMobi/InMobiAdapterConfiguration.m [hunk 2]
```diff
@@ -84,12 +86,21 @@ + (void)initializeInMobiSDK:(NSString *)accountId {
         };
         
         [InMobiAdapterConfiguration invokeOnMainThreadAsSynced:YES withCompletionBlock:completionBlock];
+        [InMobiAdapterConfiguration updatePartnerGDPRConsent];
```
Comment (p158276) [inline InMobi/InMobiAdapterConfiguration.m:89]: Disconnect the consent setting from the initialization code and call the consent logic before request to ensure consent is set irrespective of the init changes.

**Review by p158276 — CHANGES_REQUESTED** [review]

File: InMobi/InMobiAdapterConfiguration.m (hunk 2 above)
Comment (kshtgarg21) [reply InMobi/InMobiAdapterConfiguration.m:89]: Inmobi SDK also sends the GDPR consent to multiple other internal services while init is being performed hence added the consent logic here.

We are already calling consent logic before ad request.

**Comment by p158276** [comment]
Hi @kshtgarg21 ,
The changes have been reflected in this [commit](https://github.com/mopub/mopub-ios-mediation/commit/2d129d8a17e8e8221ab971ae5bfb59397dab7687) for the release. Close the PR here.

**Pull request closed without merging**